pkg/packagekit/package_fpm.go

```diff
@@ -0,0 +1,120 @@
+package packagekit
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	"github.com/pkg/errors"
+	"go.opencensus.io/trace"
+)
+
+type outputType string
+
+const (
+	Deb outputType = "deb"
+	RPM            = "rpm"
+	Tar            = "tar"
+)
+
+type fpmOptions struct {
+	outputType outputType
+}
+
+type fpmOpt func(*fpmOptions)
```

Comment:
export this type declaration, otherwise it's impossible to wrap the PackageFPM function in any way.